src/applications/facility-locator/actions/index.js

```diff
@@ -227,6 +227,7 @@ export const genBBoxFromAddress = (query) => {
               },
               bounds: minBounds,
               zoomLevel: res.features[0].id.split('.')[0] === 'region' ? 7 : 9,
+              currentPage: 1,
```

Comment:
The idea here is that if the locations list needs to be updated based on a map-change (zoom in or lat/long), then the current page should revert to 1.